examples/api_gateway/api_gateway.h

```diff
@@ -83,7 +86,7 @@ rte_atomic16_t num_running_containers;
 
 // buffer pulls from gateway and scaler ring buffers
 
-struct rte_ring *scale_buffer_ring, *gate_buffer_ring;
+struct rte_ring *scale_buffer_ring, *gate_buffer_ring, *container_init_ring;
```

Comment:
Gateway Buffer ring may not be useful.
```suggestion
struct rte_ring *scale_buffer_ring, *container_init_ring;
```